frb_rust/src/ffi/dart_api.rs

```diff
@@ -4,7 +4,16 @@
 
 use std::ffi::c_void;
 
-use dart_sys::{Dart_Handle, Dart_PersistentHandle};
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
uber-nit: should it be copy/clone? (I know we will only use the `*mut` pointer, but not sure about this derive)

Reply:
Yes, it is not necessary